src/lib/assets/mui_components/ContentCopy.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
```

Comment:
ContentCopy was not part of [@material-ui/icons](https://www.npmjs.com/package/@material-ui/icons), but it is in [@mui/icons-material](https://www.npmjs.com/package/@mui/icons-material)

Reply:
Added as a custom icon, but maybe we should move to this other icon set?

Reply:
The @mui/* packages are part of MUI v5. I'm not sure if the v5 icon pack is compatible with MUI v4 (which we're currently using as @material-ui/*)